### DISCUSSION TITLE
Wrong example, "Unresolved attribute reference".

### DISCUSSION THREAD

**Discussion by david-shiko** [fastapi-users/fastapi-users#580] [General]
Wrong example by this [link.](https://frankie567.github.io/fastapi-users/configuration/databases/sqlalchemy/#create-the-tables) Pycharm says: `Unresolved attribute reference 'metadata' for class 'DeclarativeMeta'` and `Unresolved attribute reference 'metadata' for class 'DeclarativeMeta'`

P.S I know that is mainly relating to sqlalchmey, but anyway I consider that examples should not be outdated.

**Comment by MatthewScholefield** [comment]
Hmm, can you confirm if this is actually an issue or is it just Pycharm's linter being unable to resolve dynamic references?

**Reply by david-shiko** [reply]
Unfortunatelly no because I already not using `fastaou-users`. But anyway pycharm is some kind of a default IDE for python, so I thinking it's worth to conform with it. I only hope that it's not bug of pycharm

**Reply by MatthewScholefield** [reply]
Hmm, to me it sounds like this is just a linting error of PyCharm and relates to SqlAlchemy and should be brought up their not here.
